mix.exs

```diff
@@ -54,9 +66,11 @@ defmodule Nimble.MixProject do
   # See the documentation for `Mix` for more info on aliases.
   defp aliases do
     [
+      lint: ["format --check-formatted", "credo"],
       "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
       "ecto.reset": ["ecto.drop", "ecto.setup"],
-      test: ["ecto.create --quiet", "ecto.migrate", "test"]
+      test: ["ecto.create --quiet", "ecto.migrate", "test"],
+      coverage: ["sobelow --config", "coveralls.html --raise"]
```

Comment:
Should we move `sobelow --config` to `lint` instead of `coverage` @Tolsee 🤔

Reply:
Errors raise in `sobelow` is not a `lint` issue, it's a security issue. So, for me it makes sense to put it here 🤔